modules/install/pages/install-ports.adoc

```diff
@@ -89,6 +89,11 @@ a| * Version 2 (XMEM)
 ** *Encrypted*: 11207, 18091, 18092
 
 NOTE: If enforcing TLS encryption, these ports may be blocked outside of a Couchbase Server cluster but need to remain open between nodes.
+
+| _cbbackupmgr (backup/restore client)_
+| *Unencrypted*: 8091-8096, 9102, 11210
+
+*Encrypted*: 11207, 18091-18096, 19102
```

Comment:
Add after line 96, the note below.

Note: cbbackupmgr, the backup client, connects to the Couchbase Server using the above listed ports.  You can find detailed info about the server ports in the Detailed Port Description section, but below is a summary of the the services/info cbbackupmgr is accessing via the ports:

- rest_port/ssl_rest_port :: 8091 / 18091 (Cluster admin/management)
- cbas_http_port/cbas_ssl_port :: 8095 / 18095 (Analytics service)
- memcached_port/memcached_ssl_port :: 11210 / 11207 (Data service)
- eventing_http_port / eventing_ssl_port :: 8096/18096 (Eventing service)
- indexer_http_port / indexer_https_port :: 9102 / 19102 (GSI Indexes)
- query_port / ssl_query_port :: 8093 / 18093 (Query)
- fts_http_port / fts_ssl_port :: 8094 / 18094 (Search)
- capi_port / ssl_capi_port :: 8092 / 18092 (Views)

Reply:
done